.travis.yml

```diff
@@ -1,37 +1,30 @@
 ---
-language: node_js
-node_js:
-  - "6"
-
-sudo: false
-
+language: python
+python: 3.6
 cache:
   directories:
     - $HOME/.npm
-    - $HOME/.cache # includes bowers cache
-
+    - $HOME/.cache  # includes bowers cache
 before_install:
-  - npm config set spin false
-  - npm install -g bower phantomjs-prebuilt
-  - bower --version
-  - phantomjs --version
-
-install:
+  - pip3 install coala-bears
+  - nvm install
   - npm install
+  - npm config set spin false
+  - npm install bower phantomjs-prebuilt ember-cli
+  - npm install eslint remark remark-lint remark-cli
```

Comment:
Should be in `package.json` already.

Reply:
Also please keep the spacing and everything intact, the diff is too big in this file.

Reply:
And if you do need to add dev dependencies, please do it as a separate commit.

Reply:
I will remove it.